### DISCUSSION TITLE
TPI4 - Duda tests beginTransaction

### DISCUSSION THREAD

**Discussion by jsaudino** [dds-jv/foro#386] [Consultas de persistencia] [answered]
Buenas,
estaba haciendo el TPI4 y corriendo los tests me di cuenta que nunca se hacía ningún Insert y _puedeRecuperarLugares()_ terminaba fallando.
si llamo a this.beginTransaction() antes de persistir, sí hace los insert, pero entiendo que no debería necesitar hacerlo porque de eso se encarga la clase AbstractPersistenceTest.

estoy haciendo algo mal? no toqué nada de los tests originales

**Comment by gastonprieto** [comment, marked as answer]
Buenas @jsaudino !

Fijate esta discussion: https://github.com/dds-jv/foro/discussions/313 debe ser algo de esto
